### PR TITLE
Filter by currently installed providers

### PR DESCRIPTION
**Pull request by persiaAziz**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
Only the data relating to the currently installed providers need to be displayed on the workbooks. Since the workbooks is built on top of log analaytics workspace, currently the workbooks show up all data relating to all providers that have been provisioned at least once. This change makes a correlation between the installed providers and the data to show on the workbooks. This PR only changes the HANA workbooks. There will be subsequent PRs for other provider type as well

* [x] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
Before:
before
After:
after

* [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Workbooks/SapMonitor/SapHana/SapHana.workbook [hunk 1]
```diff
@@ -2070,7 +2097,7 @@
...
...ight.lastpost\r\n;\r\n//create a table with the primary site\r\nlet table_primarysite = table_hsrlastpost \r\n| distinct  \r\n    SITE_NAME_s\r\n| project  \r\n    NodeID = SITE_NAME_s,\r\n    SourceID = '',\r\n    TargetID = '',\r\n    CentreContent = SITE_NAME_s, \r\n    EdgeLabel = '', \r\n    TopContent = 'Primary'\r\n;\r\n// create a table with the secondary sites\r\nlet table_secondarysite = table_hsrlastpost \r\n| summarize\r\n   TIME_DIFF_SECONDS_d = avg(TIME_DIFF_SECONDS_d),\r\n   OPERATION_MODE_s = max(OPERATION_MODE_s),\r\n   REPLICATION_MODE_s = max(REPLICATION_MODE_s),\r\n   SYSTEM_REPLICATION_STATUS_s = max(SYSTEM_REPLICATION_STATUS_s)\r\n  by\r\n  SITE_NAME_s,\r\n   SECONDARY_SITE_NAME_s\r\n| project  \r\n   NodeID = SECONDARY_SITE_NAME_s,\r\n   SourceID = SITE_NAME_s,\r\n   TargetID = SECONDARY_SITE_NAME_s,\r\n   CentreContent = SECONDARY_SITE_NAME_s,\r\n   EdgeLabel = strcat( iif(SYSTEM_REPLICATION_STATUS_s == 'ACTIVE', 'ðŸŸ¢', 'ðŸŸ¥'), ' ', SYSTEM_REPLICATION_STATUS_s, ' - ', REPLICATION_MODE_s, ' with ',OPERATION_MODE_s), \r\n   TopContent = 'Secondary'\r\n;\r\n// join the two together\r\ntable_primarysite\r\n| union table_secondarysite\r\n| order by NodeID asc",
```
Comment (gardnerjr) [inline Workbooks/SapMonitor/SapHana/SapHana.workbook:2100]: not sure if this is an issue in the workbook, or github's diff tool or what.  in the before, i see:
image
in the after i see:
`'ACTIVE', 'ðŸŸ¢', 'ðŸŸ¥')`

**Review by shaaga — APPROVED** [review]

File: Workbooks/SapMonitor/SapHana/SapHana.workbook (hunk 1 above)
Comment (persiaAziz) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook:2100]: Huh. I haven't touched that at all. Any suggestion on what I can do here?

**Review by PakDLiu — APPROVED** [review]

File: Workbooks/SapMonitor/SapHana/SapHana.workbook (hunk 1 above)
Comment (tniek) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook:2100]: @persiaAziz use a text editor that is unicode capable so you don't use the emojis 🔥

**Review by tniek — COMMENTED** [review]
Careful on the unicode emojis!

File: Workbooks/SapMonitor/SapHana/SapHana.workbook (hunk 1 above)
Comment (persiaAziz) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook:2100]: I just copied the template from the portal directly and pasted here. The template on the portal does not contain the green and red shapes .  What editor do you use?

Comment (persiaAziz) [reply Workbooks/SapMonitor/SapHana/SapHana.workbook]: okay fixed  it

**Review by tniek — APPROVED** [review]
Thank you

**Pull request merged**